packages/shared/src/loaders/deriv-components-loader.js

```diff
@@ -0,0 +1,21 @@
+// import { Button } from 'deriv-components';
+//  ↓ ↓ ↓
+// import Button from 'deriv-components/lib/button';
+// import 'deriv-components/lib/button.css';
```

Comment:
Would you please remove these comments and add a short one on what this component does?